src/api/services/participantsService.ts

```diff
@@ -127,13 +127,15 @@ const updateParticipantAssociatedRequestTypes = async (
   await participant.$relatedQuery('types', trx).relate(participantApprovalPartial.types);
 };
 
-const updateParticipantAssociatedRequestApiRoles = async (
+export const updateParticipantAssociatedRequestApiRoles = async (
```

Comment:
This name is confusing me now, can it just be `updateParticipantApiRoles`?

Reply:
I assume it was named this in reference to the fact that it's doing the update as part of a large db transaction. I'm not sure renaming it would be for the better as it woudl ignore the fact that a transaction has to be passed in.

Reply:
Renamed the function (`updateParticipantApiRolesUsingTransaction`) and moved the logic that was in the router to a new function in service (`updateParticipantApiRoles`).